geniza/corpus/management/commands/escr_alto_to_annotation.py

```diff
@@ -232,33 +272,32 @@ def scale_polygon(self, polygon, scale):
         # return as string for use in svg polygon element
         return " ".join([str(point) for point in scaled_points])
 
-    def create_block_annotation(self, textblock, canvas_uri, scale_factor):
+    def create_block_annotation(
+        self, textblock, canvas_uri, scale_factor, block_type, order
+    ):
         """Produce a valid IIIF annotation with the block-level content and geometry,
         linked to the IIIF canvas by URI"""
 
-        # lines to HTML list
-        block_text = "<ol>\n"
-        for line in textblock.lines:
-            block_text += f"<li>{line.content}</li>\n"
-        block_text += "</ol>"
-
         # create IIIF annotation
         anno_content = {}
-        anno_content["body"] = [
-            {
-                "TextInput": "rtl",
-                "format": "text/html",
-                "type": "TextualBody",
-                "value": block_text,
-            }
-        ]
+        anno_content["schema:position"] = order
+        anno_content["textGranularity"] = "block"
         anno_content["motivation"] = ["sc:supplementing", "transcribing"]
         anno_content["target"] = {
             "source": {
                 "id": canvas_uri,
                 "type": "Canvas",
             },
         }
+        if block_type:
+            anno_content["body"] = [
+                {
+                    "label": block_type,
+                }
+            ]
+            if block_type in self.rotation_tags:
+                # add rotation tag as a CSS class to this block
+                anno_content["target"]["styleClass"] = block_type
```

Comment:
Not using it for anything yet, but figured I would include it in the ingest code so we don't have to make modifications later to store this info. Eventually it will be used for showing the rotated lines in the line-by-line editor.